_targets/ia32/generic/build.project

```diff
@@ -33,6 +33,7 @@ export PORTS_OPENVPN=y
 export PORTS_JANSSON=y
 export PORTS_CURL=y
 export PORTS_MICROPYTHON=y
+export PORTS_LSB_VSX=y
```

Comment:
Let's place it in _projects/ia32-generic-qemu as you didn't test it on `ia32-generic-pc`

Reply:
Was at least one port tested on ia32-generic-pc?

I think keeping it in 1 place is more convenient and error prone

Reply:
I am not sure, but if not - they shouldn't be built in my opinion. However, I see that some other new ports like `lua` are placed in `_targets`, so I am leaving the decision to @Darchiv.

Reply:
Well, most of the ports are put in `_targets`, as they mainly depend on the hardware architecture itself and not low-level platform details (peripherals, etc.). I think it is fine to put it in `_targets` now.

FYI: We may need to improve how ports are built as a whole and this use case should also be considered.